maintenance/passport-azure-ad/lib/aadutils.js

```diff
@@ -253,66 +246,80 @@ exports.getErrorMessage = (err) => {
 };
 
 exports.concatUrl = (url, rest) => {
-  if (typeof rest === 'string' || rest instanceof String) {
-    rest = [rest];
+  let validRest;
+  if (typeof rest === "string" || rest instanceof String) {
+    validRest = [rest];
+  } else {
+    validRest = rest;
   }
 
   if (!url) {
-    return `?${rest.join('&')}`;
+    return `?${validRest.join("&")}`;
   }
 
-  var hasParam = url.indexOf('?') !== -1;
-  return rest ? url.concat(hasParam ? '&' : '?').concat(rest.join('&')) : url;
+  const hasParam = url.indexOf("?") !== -1;
+  return validRest ? url.concat(hasParam ? "&" : "?").concat(validRest.join("&")) : url;
 };
 
-
-// This is a list maintained by the AAD server team, will need to keep an eye on this
-// as things change.  Not ideal, but it is what it is
-// in general, a change like this (adding a new cookie attribute)
-// should be backward compatible as RFC 6265 specifies that browsers should
-// ignore unknown cookie attributes. However, for the specific case of the
-// SameSite attribute, some browsers incorrectly implemented the attribute or
-//  implemented an earlier draft which had contradictory behavior.
-// For these browsers which attempted to support SameSite,
-// but have bugs in their support, we want to omit the SameSite=None attribute.
-// See Chromium official guidance here:  https://www.chromium.org/updates/same-site/incompatible-clients
+/*
+ * This is a list maintained by the AAD server team, will need to keep an eye on this
+ * as things change.  Not ideal, but it is what it is
```

Comment:
"Not ideal, but it is what it is"

I realize text is is the older iteration, but can we remove opinions on implementations here?

Reply:
Noted. Opinions removed on PR #3996